Task1/intro.json

```diff
@@ -1,238 +1,243 @@
+		"role": ["contributor"]
+	},
+	{
+		"name": "Abdullahi Yunus",
+		"github_handle": "Abdulkbk",
+		"hacktoberfest_round": "first",
+		"role": ["contributor"]
+	},
+	{
+		"name": "Renan Miranda",
+		"github_handle": "renanlmiranda",
+		"hacktoberfest_round": "second",
+		"role": ["contributor"]
+	},
+	{
+		"name": "Shruti Mohanty",
+		"github_handle": "shruti0419",
+		"hacktoberfest_round": "first",
+		"role": ["contributor"]
+	},
+	{
+		"name": "Sudipta Pradhan",
+		"github_handle": "sudiptapradhan",
+		"hacktoberfest_round": "first",
+		"role": ["contributor"]
+	},
+	{
+		"name": "Manjusha Iyer",
+		"github_handle": "manjusha2912",
+		"hacktoberfest_round": "first",
+		"role": ["contributor"]
+	},
+	{
+		"name": "Aravindh Sivalingam",
+		"github_handle": "Aravindhsiva",
+		"hacktoberfest_round": "first",
+		"role": ["contributor"]
+	},
+	{
+		"name": "Gourav Chawla",
+		"github_handle": "gouravchawla334",
+		"hacktoberfest_round": "first",
+		"role": ["contributor"]
+	},
+	{
+		"name": "Nishant Ola",
+		"github_handle": "ola-nishant",
+		"hacktoberfest_round": "first",
+		"role": ["contributor"]
+	},
+	{
+		"name": "Abongwa Bonalais Amahnui",
+		"github_handle": "Burnleydev1",
+		"hacktoberfest_round": "third",
+		"role": ["contributor", "Maintainer"]
+	},
+	{
+		"name": "Krishnapal Mori ",
+		"github_handle": "Krishnapalmori301",
+		"hacktoberfest_round": "first",
+		"role": ["contributor"]
+	},
+	{
+		"name": "Ndi Lionel",
+		"github_handle": "spykelionel",
+		"hacktoberfest_round": "second",
+		"role": ["contributor", "maintainer"]
+	},
+	{
+		"name": "Lando Momo Paul",
+		"github_handle": "lando-paul",
+		"hacktoberfest_round": "first",
+		"role": ["contributor"]
+	},
+	{
+		"name": "Krishnapal Mori ",
+		"github_handle": "Krishnapalmori301",
+		"hacktoberfest_round": "first",
+		"role": ["contributor"]
+	},
+	{
+		"name": "Dhruv Gupta",
+		"github_handle": "MR-DHRUV",
+		"hacktoberfest_round": "first",
+		"role": ["contributor"]
+	},
+	{
+		"name": "Sonia Amehmbo",
+		"github_handle": "xennia316",
+		"hacktoberfest_round": "first",
+		"role": ["maintainer", "contributor"]
+	},
+	{
+		"name": "Aman Fangeria",
+		"github_handle": "amanfangeria980",
+		"hacktoberfest_round": "first",
+		"role": "contributor"
+	},
+	{
+		"name": "Dibyajit Tripathy",
+		"github_handle": "DibyajitTripathy",
+		"hacktoberfest_round": "third",
+		"role": ["contributor"]
+	},
+	{
+		"name": "Baimam Boukar Jean Jacques",
+		"github_handle": "baimamboukar",
+		"hacktoberfest_round": "second",
+		"role": ["contributor", "maintainer"]
+	},
+	{
+		"name": "Kshitij Verma",
+		"github_handle": "kshitijv256",
+		"hacktoberfest_round": "first",
+		"role": ["contributor"]
+	},
+	{
+		"name": "Nfon Andrew",
+		"github_handle": "andrew21-mch",
+		"hacktoberfest_round": "third",
+		"role": ["contributor"]
+	},
 
-  },
-  {
-    "name": "Azoh Munoh",
-    "github_handle": "azohMu",
-    "hacktoberfest_round": "first",
-    "role": ["contributor"]
-  },

```

Comment:
[Check this comment](https://github.com/Developer-Student-Clubs-UBa/hactoberfest2022/pull/159#issuecomment-1264360617)
Please don't delete other members' intro. ONLY add yours